aten/src/ATen/FunctionalizeFallbackKernel.cpp

```diff
@@ -320,11 +320,9 @@ static at::Tensor _unsafe_view_functionalize(const at::Tensor & self, at::SymInt
   auto stride = at::detail::computeStride(self.sym_sizes(), self.sym_strides(), inferred_size);
 
   if (!stride.has_value()) {
-    // With unbacked symints, computeStride could fail even on contiguous
-    // tensors. In this case, we can use the strides of an empty tensor of
-    // inferred_size.
+
     TORCH_CHECK(
-        self.is_contiguous(),
+        self.is_contiguous_or_false(),
```

Comment:
Not going to decompose this into `guard_or_false(self.sym_is_contiguous())`?

Reply:
In fact, because this is a TORCH_CHECK, it would be optimal to do `TORCH_SYM_CHECK(self.sym_is_contiguous())` and this too is guaranteed not to DDE.

Reply:
addressed  make sense.